application/application_container.go

```diff
@@ -0,0 +1,358 @@
+package application
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/fatih/structs"
+	"github.com/hashicorp/go-oracle-terraform/client"
+	"strings"
+)
+
+const WaitForApplicationContainerRunningTimeout = time.Duration(600 * time.Second)
```

Comment:
I'd say yes. It's easier to change at the top of the file. Right now they are just used in one place but it could change so it's easier to keep/ change here